### PR TITLE
Move Ventura support in scripts/install.sh to scripts/build.sh; fix for node build on ventura

### PR DESCRIPTION
**Pull request by eatonphil**
## Pre-merge checklist

* [x] I am very sure this PR could not affect performance.

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -8,6 +8,20 @@ else
...
+	target_set="true"
+	;;
+esac
+
+if [ "$target_set" = "false" ]; then
+    if [ "$(./zig/zig targets | grep '"os": "' | cut -d ":" -f 2 | cut -d '"' -f 2)" = "macos" ]; then
+	# Add target to args
+	set -- "$@" "-Dtarget=native-macos"
+    fi
+fi
```
Comment (kprotty) [inline scripts/build.sh]: Moving the target check into build.sh makes more sense over requiring it externally. Could the target flag also be removed in `benchmark.sh` now?

Comment (eatonphil) [reply scripts/build.sh]: Ok, I gave that a shot too!

**Comment by cb22** [comment]
@eatonphil looks like the mac benchmark is broken?

**Comment by eatonphil** [comment]
@kprotty @cb22 I dropped the benchmark.sh fix for now so we can get the build.sh in since there was an issue in benchmark.sh I didn't have time to look at.

**Review by cb22 — APPROVED** [review]

**Pull request merged**